### ISSUE TITLE
Territory owners should be able to post in their territories for free

### ISSUE THREAD

**Issue by NEEDcreations** [stackernews/stacker.news#787]
We are responsible for trying to promote our territories with posts and we pay 100k sats to SN each month. One of the perks should be that we get to post for free in our own territories (with no 10x increase within 10 minutes too). Ain't nobody got time to wait 10 minutes when trying to bring engagement to their territory, The owners aren't going to spam their own territories with pointless stuff, so we don't need to be rate limited.

**Comment by huumn** [comment]
I don't think it should be free, but I do agree the posting cost shouldn't escalate. We do this already for the comments of OPs so it would be consistent with that.
